source/contributing.rst

```diff
@@ -0,0 +1,60 @@
+.. index:: ! contributing 
+
+Contributing
+############
+
+**If you want to contribute to Rsyslog code** (thank you!), start having a look at overview and learning some basic concepts before getting to work with issues or enhacements. Joining the `mail list <http://lists.adiscon.net/mailman/listinfo/rsyslog>`_ is also a good idea.
+
+**If you want to contribute to Rsyslog documentation** (thank you!) you should also start having a look at overview and learning some basic concepts, and also you'll need to know how documentation is built.
+
+As you may know Rsyslog documentation is done using `reStructuredText <http://docutils.sourceforge.net/rst.html>`_ and 
+built with `Sphinx <http://sphinx-doc.org/contents.html>`_ to be published as HTML 
+in `rsyslog.com <http://www.rsyslog.com/doc/master/>`_.
+
+**These are the required steps to build the documentation**
+
+1. Install required software:
+
+    python, git
```

Comment:
add sphinx to this line rather than giving instructions on installing pip and then installing sphinx through pip. Many (most??) distros will have sphinx as a package and I believe that it's better to have all the packages managed by one tool so that they all get upgraded together rather than using different tools for different packages. I reserve the use of pip (and cpan and ...) to times when the packages aren't available or aren't up-to-date enough